src/features/auth/hooks/useHandleForbidden.ts

```diff
@@ -0,0 +1,21 @@
+import { AxiosError } from 'axios'
+import { useRouter } from 'next/router'
+import { noAuthRoute } from '@features/auth/utils/noAuthRoute'
+import { StatusCode } from '@utils/StatusCode'
+
+export const useHandleForbidden = (error?: AxiosError) => {
+  const router = useRouter()
+
+  if (!error) return
+
+  const statusCodesMeaningForbidden = [
+    StatusCode.FORBIDDEN,
+    StatusCode.UNAUTHORIZED
+  ]
+
+  const statusCode = Number(error.status)
+
+  const isForbidden = statusCodesMeaningForbidden.includes(statusCode)
+
+  if (isForbidden) router.push(noAuthRoute())
```

Comment:
This redirects user to home page and doesn't indicate anything about being unauthorized or not having sufficient permission. You should redirect user to a 401 page or to a 403 page.